### ISSUE TITLE
Build SDK breaking change between .net 3.1.201 and 3.1.300

### ISSUE THREAD

**Issue by jhudsoncedaron** [dotnet/sdk#11752]
LoginSAML.cs(374,44): error CS1069: The type name 'SignedXml' could not be found in the namespace 'System.Security.Cryptography.Xml'. This type has been forwarded to assembly 'System.Security.Cryptography.Xml, Version=4.0.3.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' Consider adding a reference to that assembly. [C:\Users\jhudson\dev\cento_forms\src\Cedaron.Login.SAML\Cedaron.Login.SAML.csproj]

But it builds with .NET SDK 3.1.201

<Project Sdk="Microsoft.NET.Sdk.Web">

      <PropertyGroup>
        <TargetFramework>netcoreapp3.1</TargetFramework>
        <RuntimeFrameworkVersion>3.1.3</RuntimeFrameworkVersion>
        <LangVersion>8.0</LangVersion>
        <OutputType>Exe</OutputType>
        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
      </PropertyGroup>
    
      <ItemGroup>
        <Compile Include="../AssemblyInfo.cs" />
      </ItemGroup>
    
      <ItemGroup>
        <PackageReference Include="Cedaron.Common" Version="2.0.0.6" />
        <PackageReference Include="Emet.FileSystems" Version="0.0.2" />
        <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
      </ItemGroup>
      <ItemGroup>
        <ProjectReference Include="..\Cedaron.Login.Token\Cedaron.Login.Token.csproj" />
        <ProjectReference Include="..\Cedaron.Install.Config\Cedaron.Install.Config.csproj" />
      </ItemGroup>
    </Project>

**Comment by joeloff** [comment]
3.1.300 includes the 3.1.4 runtime. 3.1.201 ships with the 3.1.3 runtime. You appear to be targeting 3.1.3 specifically. Does the machine have both SDKs installed or are you building on two different machines?

**Comment by jhudsoncedaron** [comment]
Yes both SDKs are installed. The newer runtime was installed by a Visual Studio update but are release target is still 3.1.3 and only 3.1.201 is on the build server.

**Comment by joeloff** [comment]
I did track down a known issue: https://github.com/dotnet/core/blob/master/release-notes/3.1/3.1-known-issues.md#net-core-312-sdk-31102-sdk

There's a suggestion to add an explicit package reference: 
```
<ItemGroup>
    <PackageReference Include="System.Security.Cryptography.Xml" />
</ItemGroup>
```

**Comment by jhudsoncedaron** [comment]
Same issue. I still haven't found another sdk issue number for it.

**Comment by joeloff** [comment]
I'll try and get an answer for you. I'm busy setting up a repro machine on my end.
